experiment/schema.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "namespace": "tabsplit",
+    "description": "Tab Split",
+    "permissions": ["experiments.tabsplit"],
+    "functions": [
```

Comment:
API-wise, obviously it's best to start with the simple monolithic API. I'm guessing ultimately an API might look like:

- `init` would be implicit (i.e., it would always do the right thing, maybe lazily initialize on first use of the API?)
- `tabsplit` might take two tabIds (let the WebExtension create the new tab)
- `onTabUnsplit.addEventListener()` for detecting when that happens
- `unsplitAllTabs()`
- `setTabRatio` for resizing

Then the popup becomes a WebExtension feature. One of our top Stretch goals is the ability to move an existing tab into a group via a newtab replacement, which would imply some other API to that effect.

That said, we shouldn't do more than is necessary for the product goals, so it's just a matter of what's easiest.

The mobile User-Agent modifications are best done by targeting a specific tabId, so if the API lets the WebExtension track which tabs are being split that would be very helpful. Also if we do User-Agent switching via a pref in the panel, it would be great if the panel was part of the WebExtension.